### DISCUSSION TITLE
have a option to not show recent folder history

### DISCUSSION THREAD

**Discussion by killertofus** [linuxmint/discussions#622] [Ideas]
for privacy

**Comment by rcalixte** [comment]
I believe this is already a setting under the Privacy section.

![image](https://github.com/user-attachments/assets/3f731501-3036-448f-9b37-04d953747d82)

This switch would just be set to off:

![image](https://github.com/user-attachments/assets/83cee9e6-81d3-4c10-89dd-78c32bd17805)

**Comment by killertofus** [comment]
@rcalixte that works if you are using cinnamon, if you are not using cinnamon, that settings is not there

**Reply by rcalixte** [reply]
> @rcalixte that works if you are using cinnamon, if you are not using cinnamon, that settings is not there

Are you asking for this specifically for Nemo? Something else? If it's Nemo, you can adjust the setting without Cinnamon if you run `gsettings set org.cinnamon.desktop.privacy remember-recent-files false` in a terminal.
